plugins/base/src/test/kotlin/enums/EnumsTest.kt

```diff
@@ -98,11 +288,11 @@ class EnumsTest : BaseAbstractTest() {
 
         testInline(
             """
-            |/src/main/kotlin/basic/Test.kt
-            |package enums
+            |/src/main/kotlin/basic/TestEnum.kt
+            |package testpackage
```

Comment:
`Test` and `enums` were rather generic and caused a bit of confusion. Decided to rename it